.github/workflows/tests.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10-dev]
+        python-version: [pypy-3.6, 3.6, 3.7, 3.8, 3.9, 3.10-dev]
```

Comment:
```suggestion
        python-version: [pypy-3.6, 3.6, 3.7, pypy-3.7, 3.8, 3.9, 3.10-dev]
```

Reply:
Thanks, do you know about GitHub CIs usage limits? If there's no risk of running out then this is great

Reply:
you can call them up all q.q like and get more minutes

Reply:
Haha good to know